gbpservice/nfp/service_vendor_agents/vyos/vyos_session/utils.py

```diff
@@ -68,6 +69,13 @@ def init_logger(logger):
     # add the handlers to the logger
     logger.addHandler(file_handler)
 
+    formatter = logging.Formatter('vyos %(name)s %(funcName)s() %(levelname)s '
+                                  '%(message)s')
```

Comment:
This facilitates agent logs to be registered with syslog.

Regards,
Deepak

On Mon, May 2, 2016 at 3:29 PM, njagadish <notifications@github.com> wrote:

> In gbpservice/nfp/service_vendor_agents/vyos/vyos_session/utils.py
> <https://github.com/oneconvergence/group-based-policy/pull/272#discussion_r61722203>
> :
>
> > @@ -68,6 +69,13 @@ def init_logger(logger):
> >      # add the handlers to the logger
> >      logger.addHandler(file_handler)
> >
> > +    formatter = logging.Formatter('vyos %(name)s %(funcName)s() %(levelname)s '
> > +                                  '%(message)s')
>
> Is this done to make the default log forwarding enabled to local host
> irrespective of log forwarding configured or not?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/272/files/5355935019dfd5d052b87f1642c2883137a3dffa#r61722203>
>